### PR TITLE
Fix linter errors

### PR DESCRIPTION
**Pull request by DomAyre**
Run lint.sh with -f

### CODE REVIEW

File: src/endpoints/proto/gen/hpke_pb.ts [hunk 1]
```diff
@@ -19,7 +19,14 @@
 /* eslint-disable */
```
Comment (beejones) [inline src/endpoints/proto/gen/hpke_pb.ts:19]: we shouldn't touch machine generated files.

File: src/endpoints/proto/gen/tink_pb.ts [hunk 2]
```diff
@@ -21,7 +21,14 @@
 /* eslint-disable */
```
Comment (beejones) [inline src/endpoints/proto/gen/tink_pb.ts:21]: same here

File: src/endpoints/proto/gen/hpke_pb.ts (hunk 1 above)
Comment (DomAyre) [reply src/endpoints/proto/gen/hpke_pb.ts:19]: Should we then update lint.sh to ignore them?

Comment (DomAyre) [reply src/endpoints/proto/gen/hpke_pb.ts:19]: Done in https://github.com/microsoft/azure-privacy-sandbox-kms/pull/11/commits/c19596e2ee3530eb66d4de78742d2bc0daec1302

Comment (beejones) [reply src/endpoints/proto/gen/hpke_pb.ts:19]: they already have ignore tags but clearly these tags are ignored :smile:. There is a way to configure lint. I can have a look to that after the JWT work or if you want to look into that, that's ok for me.

Comment (DomAyre) [reply src/endpoints/proto/gen/hpke_pb.ts]: I had a look into it, looks like the ignore tags are for tools other than prettier, so I think the simplest solution is to ignore files in the `/gen` directory which is also conceptually neat. 

As of https://github.com/microsoft/azure-privacy-sandbox-kms/pull/11/commits/d4393c4df8e66c6e171f42b4edd2195c0b09db91 the linter passes without any changes to the generated files, so I think we're good to merge this if you're happy

File: src/endpoints/proto/gen/tink_pb.ts (hunk 2 above)
Comment (DomAyre) [reply src/endpoints/proto/gen/tink_pb.ts]: Discussion above

**Review by beejones — APPROVED** [review]

**Pull request merged**